Makefile

```diff
@@ -1,5 +1,6 @@
 # Variables are documented in common/build.sh.
 BASE_IMAGE_NAME = ruby
+TEST_SUITES=test/run-conu
```

Comment:
This would effectively disable `test/run` test suite, right?

Reply:
I'm actually really fuzzy about your scripts and most of the time I'm like
![no idea what I'm doing.](https://media.giphy.com/media/VXCPgZwEP7f1e/giphy.gif)

So yeah, it seems it disables it. Originally I wanted to move it to $ROOT/test but I realized I don't know how to do it.

Reply:
I think it is ok the way it is. Just change the assignment to append the new test suite instead and move it so that it is appended after `common.mk` is included